### ISSUE TITLE
Custom unit appears only in iterations/second count

### ISSUE THREAD

**Issue by LoneWanderer-GH** [tqdm/tqdm#952]
- [X] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [X] visual output bug
    + [ ] documentation request (i.e. "X is missing from the documentation." If instead I want to ask "how to use X?" I understand [StackOverflow#tqdm] is more appropriate)
    + [ ] new feature request
- [X] I have visited the [source website], and in particular
  read the [known issues]
- [X] I have searched through the [issue tracker] for duplicates
- [X] I have mentioned version numbers, operating system and
  environment, where applicable:
  ```python
  import tqdm, sys
  print(tqdm.__version__, sys.version, sys.platform)
  ```

```
4.45.0
3.7.5 (default, Oct 31 2019, 15:18:51) [MSC v.1916 64 bit (AMD64)]
win32 ## (but its a win10 x64 machine)
```

While reading https://github.com/tqdm/tqdm/issues/765 I noticed the following minor thing.

with TQDM 4.45.0, when I write some downloading code, I try to use IEEE1541 units.

So, I wrote this:
```
with tqdm(total=total_size, unit_scale=True, unit_divisor=1024, unit="iB") as pbar, open(file_path, "wb") as f:
        pbar.set_description("Download {}".format(file_data["full_url"]))
        def cb(data):
            l = len(data)
            pbar.update(l)
            f.write(data)
```

Then I get:
```Download <stuff>: 100%|██████████| 34.7M/34.7M [01:14<00:00, 491kiB/s]```

I'm a bit rigid about units, so I expected it to be:

```Download <stuff>: 100%|██████████| 34.7MiB/34.7MiB [01:14<00:00, 491kiB/s]```

For emphasis:
```34.7M/34.7M``` should become
```34.7MiB/34.7MiB``` (since I set ```unit="iB"```)

The iterator part correctly takes into account the unit suffix ```491kiB/s```, but not the current/total size part.

(yes, in a way, I'm trying be an evangelist of IEEE 1541 standard (see https://en.wikipedia.org/wiki/Mebibyte). You would have a much bigger problem handling the case difference for kilos (**`k`** in decimal, but **`K`** in byte units ... )

  [source website]: https://github.com/tqdm/tqdm/
  [known issues]: https://github.com/tqdm/tqdm/#faq-and-known-issues
  [issue tracker]: https://github.com/tqdm/tqdm/issues?q=
  [StackOverflow#tqdm]: https://stackoverflow.com/questions/tagged/tqdm

**Comment by casperdcl** [comment]
The units are intentionally not repeated 3 times in order to leave more room for display of other things. This is customisable to your own preferences using `bar_format`, i.e.:

```python
tqdm(...,
     bar_format="{l_bar}{bar}| {n_fmt}{unit}/{total_fmt}{unit}"
     " [{elapsed}<{remaining}, {rate_fmt}{postfix}]")
```

If you want to go further down the road of taking up more space in the name of conforming to a specification you may even want to add some spaces vis #825.

**Comment by casperdcl** [comment]
I think this is all handled in #1063. Note that:

- technically the prefix is `Ki` while the unit is `B`
- `12.7 KiB/34.7 MiB` is a division therefore the units can cancel and is equivalent to `12.7 Ki/34.7 Mi` which takes up less space while still being technically correct

**Comment by almson** [comment]
I agree I'd like to see the units at least twice in the bar. I'm not sure why we want narrow bars.
